include/RE/Offsets.h

```diff
@@ -651,6 +651,7 @@ namespace RE
 		{
 			inline constexpr REL::ID EnsureQuestStarted(static_cast<std::uint64_t>(24481));
 			inline constexpr REL::ID ResetQuest(static_cast<std::uint64_t>(24486));
+			inline constexpr REL::ID ForceRefIntoAliasID(static_cast<std::uint64_t>(24523));
```

Comment:
`Offsets.h` is deprecated, please don't use it.

Reply:
What's the replacement for this file?

Reply:
There is no replacement, just define it ad-hoc